<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="74535" started-at="2020-05-20T16:52:25Z" finished-at="2020-05-20T16:53:39Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="74535" started-at="2020-05-20T16:52:25Z" finished-at="2020-05-20T16:53:39Z">
      <class name="com.qa.testcases.Assignment2">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.Assignment2@7cf10a6f]" name="setUp" is-config="true" duration-ms="54538" started-at="2020-05-20T22:22:25Z" finished-at="2020-05-20T22:23:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="VerificationDatePickerTest()[pri:0, instance:com.qa.testcases.Assignment2@7cf10a6f]" name="VerificationDatePickerTest" duration-ms="12957" started-at="2020-05-20T22:23:19Z" finished-at="2020-05-20T22:23:32Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.qa.pages.RegisterPage.RegisterwithFields(RegisterPage.java:86)
	at com.qa.testcases.Assignment2.VerificationDatePickerTest(Assignment2.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerificationDatePickerTest -->
        <test-method status="PASS" signature="actionaftrtest()[pri:0, instance:com.qa.testcases.Assignment2@7cf10a6f]" name="actionaftrtest" is-config="true" duration-ms="6322" started-at="2020-05-20T22:23:33Z" finished-at="2020-05-20T22:23:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- actionaftrtest -->
      </class> <!-- com.qa.testcases.Assignment2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
